docs/source/traitsui_user_manual/advanced_view.rst

```diff
@@ -118,7 +96,7 @@ example above would be implemented as follows:
 ::
 
     # default_traits_view2.py -- Sample code to demonstrate the use of
```

Comment:
no literal include here because this file doesn't exist I'm guessing?

Reply:
yeah i couldn't find the file and i didn't bother creating a new one. creating one now.